packages/charts/src/charts/bubble/tooltip-content.component.tsx

```diff
@@ -1,20 +1,24 @@
 import React, { FC, useMemo } from 'react';
 import { Typography } from '@keen.io/ui-core';
-import { getFromPath, ScaleSettings } from '@keen.io/charts-utils';
+import { getFromPath } from '@keen.io/charts-utils';
 
 import Correlation from './correlation.component';
-import { formatTooltipValue } from '../../utils/tooltip.utils';
 
-import { DataSelector } from '../../types';
+import { DataSelector, TooltipFormatter } from '../../types';
 
 type Props = {
   data: Record<string, any>[];
   typography: Typography;
   labelSelector: string;
-  valueKey: string;
   selectors: { selector: DataSelector; color: string }[];
-  yScaleSettings?: ScaleSettings;
-  yDomainKey?: string;
+  valueKey: string;
+  xDomainKey: string;
+  yDomainKey: string;
+  formatValue?: {
+    xKey?: TooltipFormatter;
```

Comment:
I suggest to use `Partial` here to make all object keys optional without adding `?` explicit.
```
formatValue?: Partial<{ xKey: TooltipFormatter, ... }>
```